src/config/dynamic-theme-fixes.config

```diff
@@ -23497,6 +23497,16 @@ CSS
 
 ================================
 
+typst.app
+
+CSS
```

Comment:
For some reason I can't edit the file, please use the following config instead:
```
typst.app

INVERT
img[src$=".svg"]

```

Reply:
Does `INVERT` literally add `filter: invert();`? Or is it something different?

Reply:
Yes, this is an equivalent of `invert(1)` (not sure if it works without a number in every browser). You can leave the long selectors if they work better (but for INVERT section each selector should be on a separate line without commas).

Reply:
> for INVERT section each selector should be on a separate line without commas

Yes, I've seen this before. (But I forgot about it all together and used pure CSS.)